### PR TITLE
Rw product model and view testing

### PR DESCRIPTION
**Pull request by rtwhitfield84**
## Description
In this iteration of testing, i completed the initial product model testing with passing tests asserting that the model fields were correct.
I then moved onto the product view testing and have begun the initial iteration of this process as the tests are still failing.

during this process, it was necessary to update the project urls, and the bang_app urls.
I've also created a products.html file as a placeholder for view redirection testing.

I also made migrations to include the new product and product type models

## Related Ticket(s)
#15 

## Problem to Solve
Customer should be able to create a product

## Expected Behavior
Model testing should pass
Views testing still failing

## Steps to Test Solution

```
git fetch

git checkout rw_product_testing

python manage.py makemigrations

python manage.py test bang_app
```

## Unit Testing
[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation

[ ] There is new documentation in this pull request that must be reviewed..
[x] I added documentation for any new classes/methods

### CODE REVIEW

File: bangazon_website/bang_app/views/products_view.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class ProductsView(TemplateView):
+	template_name = 'bang_app/products.html'
+
+def create_product(request):
+	new_product = request.POST
+	Product.objects.create(
+		name=new_product['product_name'],
+		description=new_product['description'],
+		price=new_product['price'],
+		quantity=new_product['quantity']
```
Comment (samphillips1879) [inline bangazon_website/bang_app/views/products_view.py]: You need a comma at the end of this line 🌁

**Comment by sales-ls21** [comment]
Tests FAIL! :+1:

**Comment by samphillips1879** [comment]
Confirm that model tests pass but view tests fail 👍

**Comment by whitneycormack** [comment]
👍

**Pull request merged**